src/neo4j_graphrag/experimental/pipeline/pipeline.py

```diff
@@ -600,20 +650,23 @@ async def run(
             result=await self.get_final_results(orchestrator.run_id),
         )
 
-    def dump_state(self, run_id: str) -> Dict[str, Any]:
+    def dump_state(self) -> Dict[str, Any]:
```

Comment:
I guess we will need the run_id as parameter here as a consequence of my first comment